### PR TITLE
Update 10_Using_GitHub_exclusively-Collaborator_Step-by-step_Guide.md

### PR DESCRIPTION
**Pull request by blaukos**
Adding some elements of definition for the word "collaborator" in the Github context.

### CODE REVIEW

File: Collaborative_writing_on_GitHub/10_Using_GitHub_exclusively-Collaborator_Step-by-step_Guide.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # 10. Using GitHub exclusively – Collaborator - Step-by-step Guide
 
-In this section, you can read a detailed, step-by-step guide about how to use only GitHub for collaboration. To demonstrate this, I created a new user called ‘Test-Collaborator-1’ and invited this user to collaborate and make a suggestion to improve a fictional piece of content. 
+In this section, you can read a detailed, step-by-step guide about how to use only GitHub for collaboration. To demonstrate this, I created a new user called ‘Test-Collaborator-1’ and invited this user to collaborate and make a suggestion to improve a fictional piece of content. "Collaborator" is a specific status in Github. We use the word in line with the status as it is defined by Github.
```
Comment (attilacsontos) [inline Collaborative_writing_on_GitHub/10_Using_GitHub_exclusively-Collaborator_Step-by-step_Guide.md]: I think Github should spelled as 'GitHub', is not is?

Comment (blaukos) [reply Collaborative_writing_on_GitHub/10_Using_GitHub_exclusively-Collaborator_Step-by-step_Guide.md]: Ah yes, absolutely. Is it possible for you to modify it?

Comment (attilacsontos) [reply Collaborative_writing_on_GitHub/10_Using_GitHub_exclusively-Collaborator_Step-by-step_Guide.md]: I am sorry, I cannot do it. Can you change it, please?

**Review by attilacsontos — CHANGES_REQUESTED** [review]
I am sorry, I cannot do it. Can you change it, please?

**Comment by attilacsontos** [comment]
This looks much better, thank you :-)

**Pull request merged**